### PR TITLE
fix(crons): Allow empty check_in_id via POST

### PR DESCRIPTION
**Pull request by evanpurkhiser**
You are not required to send a check_in_id when POSTing a check-in.

#skip-changelog

### CODE REVIEW

File: tests/integration/test_monitors.py [hunk 1]
```diff
@@ -122,21 +122,20 @@ def test_monitor_post_json_body(mini_sentry, relay):
     relay = relay(mini_sentry)
     mini_sentry.add_basic_project_config(42)
 
-    check_in = generate_check_in("my-monitor")
```
Comment (evanpurkhiser) [inline tests/integration/test_monitors.py:125]: `generate_check_in` wasn't giving us much, I just dropped it in favor of inlining the check-in body.

File: tests/integration/test_monitors.py [hunk 2]
```diff
@@ -145,7 +144,7 @@ def test_monitor_post_json_body(mini_sentry, relay):
 
     check_in = json.loads(item.get_bytes().decode())
     assert check_in == {
-        "check_in_id": "a460c25ff2554577b920fcfacae4e5eb",
+        "check_in_id": "00000000000000000000000000000000",
```
Comment (evanpurkhiser) [inline tests/integration/test_monitors.py:147]: This is the main change. It should be possible to NOT specify an ID.

**Review by jan-auer — APPROVED** [review]
LGTM. Keep in mind that now you will always have to check if the ID is NIL rather than `is not None` in python.

**Comment by evanpurkhiser** [comment]
> LGTM. Keep in mind that now you will always have to check if the ID is NIL rather than is not None in python.

Not a problem, we already were handling this: https://github.com/getsentry/sentry/blob/49f7d58744444a414aa5c78fffc748e7fe2667cb/src/sentry/monitors/consumers/monitor_consumer.py#L296-L304

**Pull request merged**